File: auth-server/src/main/java/com/github/akraskovski/auth/server/domain/model/Authority.java
package com.github.akraskovski.auth.server.domain.model;

import lombok.Getter;

/**
 * Application authorities, which can be applied to a {@link User}.
 */
public enum Authority {

    ROLE_ADMIN("Administrator"),
    ROLE_USER("User");

    @Getter
    private String description;

    Authority(final String description) {
        this.description = description;
    }
}

File: auth-server/src/main/kotlin/com/github/akraskovski/auth/server/domain/model/Authority.kt
package com.github.akraskovski.auth.server.domain.model;

/**
 * Application authorities, which can be applied to a {@link User}.
 */
enum class Authority {
    ROLE_ADMIN,
    ROLE_COMPANY_ADMIN,
    ROLE_CLIENT,
    ROLE_CARRIER,
    ROLE_FORWARDER
}
